.gitlab/source_test/linux.yml

```diff
@@ -18,6 +18,11 @@
     - inv -e rtloader.install
     - inv -e rtloader.format --raise-if-changed
     - inv -e rtloader.test
+  # Placeholder script, overridden by .linux_tests when running all go tests
+  script: [ "# Skipping go tests" ]
+
+.linux_tests:
+  stage: source_test
   script:
```

Comment:
We could move the `inv -e install-tools` here since (I think) we don't need Go tooling for rtloader. Depends on whether we might want to install tools for rtloader in the future (I don't think so)